tools/exchangePublishScript.sh

```diff
@@ -61,18 +61,18 @@ branch="-b master"
 repository="https://github.com/open-horizon/examples.git"
 
 # text file containing servies and patterns to publish
-input="examples/tools/blessedSamples.txt"
+input="/tmp/examples/tools/blessedSamples.txt"
 
 topDir=$(pwd)
 error=0
 
-git clone $branch $repository
+git clone $branch $repository /tmp/examples
```

Comment:
Can we make this more 'open-horizon' specific? Something like `/tmp/open-horizon/examples` may also want to make the root path an ENV variable, and use it everywhere you are leveraging the `/tmp` path.

Reply:
Done!